src/Sylius/Component/Archetype/Model/Archetype.php

```diff
@@ -74,8 +78,7 @@ public function __construct()
         $this->attributes = new ArrayCollection();
         $this->options = new ArrayCollection();
         $this->createdAt = new \DateTime();
-
-        parent::__construct();
+        $this->__ttConstruct();
```

Comment:
Hmm, I'm not sure __ttConstruct is readable enough.. Why not name it translatableTraitConstruct? Would be much clearer imo.